src/explore-education-statistics-common/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -214,6 +215,30 @@ const FiltersForm = (props: Props & InjectedWizardProps) => {
               />
             </Form>
           </div>
+        ) : (
+          <>
+            {stepHeading}
+            {
+              // <SummaryList noBorder>
```

Comment:
This doesn't seem right

Reply:
This was commented out as I suspect this will be a post-MVP improvement that will get suggested, and here it is half-baked.

I found myself getting carried away implementing, but figured it wasn't a necessary feature for MVP so I cleaned it up and commented it out.

Would you like me to delete my comments?

Reply:
No it's ok, you can leave it.